lib/openrtb2_3/bid.js

```diff
@@ -20,13 +20,19 @@ var Bid = function(status, nurl, adm, adid, adomain, id, price, cid, clearPrice,
 
 Bid.prototype = Object.create(RtbObject.prototype);
 
-Bid.prototype.replaceMacros = function(){
+Bid.prototype.replaceMacros = function(valuesMap){
   return new Promise(function (resolve, reject) {
+    valuesMap = valuesMap || {};
```

Comment:
No it doesn't need a ```var``` because it's an argument passed in the
function.

On Tue, Oct 13, 2015 at 3:01 PM, Stelios Savva <notifications@github.com>
wrote:

> In lib/openrtb2_3/bid.js
> <https://github.com/Avocarrot/openrtb/pull/27#discussion_r41856884>:
>
> >    return new Promise(function (resolve, reject) {
> > +    valuesMap = valuesMap || {};
>
> Does valuesMap need a var?
> var valuesMap = valuesMap || {} ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Avocarrot/openrtb/pull/27/files#r41856884>.
>



-- 
*George Eracleous* * | Founder*

*In the press:*
Avocarrot takes $2M on its march to become ‘the MoPub for native mobile ads’
<http://t.sidekickopen21.com/e1t/c/5/f18dQhb0S7lC8dDMPbW2n0x6l2B9nMJW7t5XX48r540TW63RDMY8rldM6Vd76_W56dSzCf8_1j_P02?t=http%3A%2F%2Fventurebeat.com%2F2015%2F05%2F19%2Favocarrot-takes-2m-on-its-march-to-become-the-mopub-for-native-mobile-ads%2F&si=4753554287362048&pi=e24b05df-1bdc-4b21-94aa-535a6441623a>
|
VentureBeat
How to Monetize Your App: Banner Ads Vs Native Ads vs In-App Purchases
<http://t.sidekickopen21.com/e1t/c/5/f18dQhb0S7lC8dDMPbW2n0x6l2B9nMJW7t5XX48r540TW63RDMY8rldM6Vd76_W56dSzCf8_1j_P02?t=http%3A%2F%2Fwww.forbes.com%2Fsites%2Filyapozin%2F2014%2F12%2F31%2Fhow-to-monetize-your-app-banner-ads-vs-native-ads-vs-in-app-purchases%2F&si=4753554287362048&pi=e24b05df-1bdc-4b21-94aa-535a6441623a>
|
Forbes
Avocarrot Tool Turns Banners Into Native Units
<http://t.sidekickopen21.com/e1t/c/5/f18dQhb0S7lC8dDMPbW2n0x6l2B9nMJW7t5XX48r540TW63RDMY8rldM6Vd76_W56dSzCf8_1j_P02?t=http%3A%2F%2Fadexchanger.com%2Fmobile%2Favocarrot-tool-takes-banners-and-turns-them-into-native-units%2F&si=4753554287362048&pi=e24b05df-1bdc-4b21-94aa-535a6441623a>
 | AdExchanger
And The Best Native Ad Award Goes To…
<http://t.sidekickopen21.com/e1t/c/5/f18dQhb0S7lC8dDMPbW2n0x6l2B9nMJW7t5XX48r540TW63RDMY8rldM6Vd76_W56dSzCf8_1j_P02?t=http%3A%2F%2Ftechcrunch.com%2F2015%2F02%2F22%2Fand-the-best-native-ad-award-goes-to%2F&si=4753554287362048&pi=e24b05df-1bdc-4b21-94aa-535a6441623a>
 | TechCrunch

L: www.linkedin.com/in/geracleous
T. 6504229317
<http://t.sidekickopen21.com/e1t/c/5/f18dQhb0S7lC8dDMPbW2n0x6l2B9nMJW7t5XX48r540TW63RDMY8rldM6Vd76_W56dSzCf8_1j_P02?t=http%3A%2F%2F6504229317%2F&si=4753554287362048&pi=e24b05df-1bdc-4b21-94aa-535a6441623a>
t:  @geracleous
<http://t.sidekickopen21.com/e1t/c/5/f18dQhb0S7lC8dDMPbW2n0x6l2B9nMJW7t5XX48r540TW63RDMY8rldM6Vd76_W56dSzCf8_1j_P02?t=https%3A%2F%2Ftwitter.com%2Fgeracleous&si=4753554287362048&pi=e24b05df-1bdc-4b21-94aa-535a6441623a>